app/helpers/sessions_helper.rb

```diff
@@ -9,6 +9,10 @@ def remember user
     cookies.permanent[:remember_token] = user.remember_token
   end
 
+  def current_user? user
```

Comment:
because this method's content relates to logic of user object, you should move it into model `User` respectively

Reply:
I tried moving but it failed